qt/application.qrc

```diff
@@ -1,10 +1,10 @@
 <RCC>
   <qresource prefix="/icons">
-      <file alias="alt-limit-off.svg">icons/turtle-silhouette.svg</file>
-      <file alias="alt-limit-on.svg">icons/turtle.svg</file>
-      <file alias="encrypted.svg">icons/lock.svg</file>
-      <file alias="hamburger-menu.svg">icons/hamburger-menu.svg</file>
-      <file alias="ratio.svg">icons/yin-yang.svg</file>
-      <file alias="transmission.svg">icons/transmission.svg</file>
+      <file alias="alt-limit-off.svg">../icons/turtle-silhouette.svg</file>
+      <file alias="alt-limit-on.svg">../icons/turtle.svg</file>
+      <file alias="encrypted.svg">../icons/lock.svg</file>
+      <file alias="hamburger-menu.svg">../icons/hamburger-menu.svg</file>
+      <file alias="ratio.svg">../icons/yin-yang.svg</file>
+      <file alias="transmission.svg">../icons/hicolor_apps_scalable_transmission.svg</file>
```

Comment:
Just pointing this out for other reviewers: `transmission.svg` and `hicolor_apps_scalable_transmission.svg` are different icons.

I don't have an opinion myself whether this is ok.

Reply:
The "old" (Tango-style) Qt client icon looked a bit dated anyway. I test it and the "new" one (from GTK client) is better in this regard, so fine by me.

Reply:
Feel free to resolve this thread, I can't do it myself.